libs/jwst-storage/src/storage/blobs/bucket_local_db.rs

```diff
@@ -39,30 +40,30 @@ impl BlobBucketDBStorage {
     pub async fn init_pool(database: &str) -> JwstStorageResult<Self> {
         todo!()
     }
+
+    /// TODO: can we merge with BlobDBStorage::metadata?
```

Comment:
Agree with this, have tried to extract a common trait auto implementing some common db operations here https://github.com/thorseraq/OctoBase/tree/0614/init-s3, but haven't finished generics adaption 🥰